example/postgres/index.js

```diff
@@ -32,7 +36,8 @@ async function startServer() {
     logger: console.log,
     signal: 'SIGINT',
     healthChecks: {
-      '/healthcheck': onHealthCheck
+      '/healthcheck': onHealthCheck,
```

Comment:
shouldn't `onHealthcheck` contain the check for the db? if the db isn't healthy, most probably the app won't be neither - what do you think?

Reply:
Yes, you are right, I change this.